cmd/support-perf.go

```diff
@@ -99,8 +99,10 @@ FLAGS:
   {{end}}
 
 EXAMPLES:
-  1. Run object storage, network, and drive performance tests on 'myminio' cluster:
-     {{.Prompt}} {{.HelpName}} myminio/
+  1. Upload object storage, network, and drive performance analysis for 'play' (https://play.min.io by default) to SUBNET
+     {{.Prompt}} {{.HelpName}} play
+  2. Run object storage, network, and drive performance tests on 'play' (https://play.min.io by default), save and upload to SUBNET manually
+     {{.Prompt}} {{.HelpName}} --airgap play
```

Comment:
I think we can avoid talking about `play/`  @anjalshireesh or maybe have it as a single reference.  More we mention `play/`  it kind of leads to abuse of the public server.

We can limit it here in this situation. - is the idea you want play because it is a working setup?

Reply:
@harshavardhana it's not my preference to use `play` as such. I was just trying to be consistent with what `mc support diag` shows. I can switch to `myminio` in all support sub-commands if that's preferred.

Reply:
Yeah we should use myminio instead, we can have one example with play

Reply:
Done. PTAL.